src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.Comments.cs

```diff
@@ -103,5 +104,25 @@ public void Comment_node_without_request_node_does_not_produce_diagnostics(strin
             result.GetDiagnostics().Should().BeEmpty();
 
         }
+
+        [Fact]
+        public void comment_after_request_separator_is_parsed_correctly()
+        {
+            var code = """
+                @MyRestaurantApi_HostAddress = https://localhost:7094
+
+                GET {{MyRestaurantApi_HostAddress}}/api/Contact
+
+                ###
+
+                # get a specific contact
+                """;
+
+            var result = Parse(code);
+
+            result.SyntaxTree.RootNode.DescendantNodesAndTokens().Should()
```

Comment:
Do we want this assertion to be more specific about where the comment node appears?

Reply:
Done